src/tests/ftest/security/cont_get_acl.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. 8F-30005.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+import os
+
+from cont_security_test_base import ContSecurityTestBase
+from command_utils import CommandFailure
+from avocado import fail_on
+
+
+class GetContainerACLTest(ContSecurityTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test Class Description:
+
+    Test to verify container ACL get command..
+
+    :avocado: recursive
+    """
+    def setUp(self):
+        """Set up each test case."""
+        super(GetContainerACLTest, self).setUp()
+        self.acl_filename = "test_acl_file.txt"
+        self.daos_cmd = self.get_daos_command()
+        self.prepare_pool()
+        self.add_container(self.pool)
+
+        # Get list of ACL entries
+        self.cont_acl = self.get_container_acl_list(
+            self.pool.uuid, self.pool.svc_ranks[0], self.container.uuid)
+
+    def error_handling(self, results, err_msg):
```

Comment:
I see this method and acl_file_diff in other test too. You might want to move them to ContSecurityTestBase.

Reply:
Will do.